sim/team-validator.ts

```diff
@@ -309,8 +309,8 @@ export class TeamValidator {
 		set.nature = dex.getNature(Dex.getString(set.nature)).name;
 		if (!Array.isArray(set.moves)) set.moves = [];
 
-		const maxLevel = format.maxLevel || 100;
-		const maxForcedLevel = format.maxForcedLevel || maxLevel;
+		const maxLevel = this.ruleTable.maxLevel || format.maxLevel || 100;
```

Comment:
```suggestion
		const maxLevel = ruleTable.maxLevel;
```

I would implement the fallbacks to `format.maxLevel` and `100` inside `ruleTable.maxLevel`, rather than out here.

Reply:
I'm sorry, I don't know how to work with classes well enough to do that

Reply:
Although it makes sense to at least have the `100` fallback out here, so you can detect _conflicting_ `maxLevel` rules.

Reply:
That's fair.